config/namelist.config

```diff
@@ -1,77 +1,133 @@
+! ============================================================================
+! This file contains the main configuration parameters for FESOM2, including:
+! - Model identification and run settings
+! - Time stepping and simulation duration
+! - Initial time/date settings
+! - File paths for mesh, forcing, and output
+! - Restart and logging configuration
+! - Vertical coordinate system (ALE)
+! - Grid geometry and rotation
+! - Calendar settings
+! - Model components (ice, cavities, etc.)
+! - Parallel decomposition
+! - Iceberg settings
+! ============================================================================
 
+! ============================================================================
+! RUN IDENTIFICATION
+! ============================================================================
 &modelname
-runid='fesom'
+runid = 'fesom'        ! run identifier (used in output filenames)
 /
 
+! ============================================================================
+! TIME STEPPING AND RUN LENGTH
+! ============================================================================
 &timestep
-step_per_day=32 !96 !96 !72 !72 !45 !72 !96
-run_length=62 !62 !62 !62 !28
-run_length_unit='y' ! y, m, d, s
+step_per_day      = 96         ! number of time steps per day (determines dt = 86400/step_per_day seconds)
+                                ! common values: 32 (45min), 48 (30min), 72 (20min), 96 (15min), 192 (7min 30sec), 
+                                ! 216 (6min 40sec), 240 (6min), 288 (5min), 360 (4min), 720 (2min), 1440 (1min), 2880 (30sec)
+run_length        = 1          ! total length of simulation run
+run_length_unit   = 'm'        ! unit for run_length: 'y' (years), 'm' (months), 'd' (days), 's' (seconds)
 /
 
-&clockinit              ! the model starts at
-timenew=0.0
-daynew=1
-yearnew=1948
+! ============================================================================
+! INITIAL TIME/DATE SETTINGS
+! ============================================================================
+&clockinit
+timenew = 0.0                  ! initial time within the day [seconds] (0.0 = midnight)
+daynew  = 1                    ! initial day of the month (1-31)
+yearnew = 1979                 ! initial year
 /
 
+! ============================================================================
+! MESH, INITIALIZATION & OUTPUT PATHS
+! ============================================================================
 &paths
-MeshPath='/pool/data/AWICM/FESOM2/MESHES/CORE2/'
-ClimateDataPath='/pool/data/AWICM/FESOM2/MESHES/INITIAL/phc3.0/'
-ResultPath='/work/ab0246/$USER/runtime/fesom2/'
+MeshPath         = '/albedo/pool/FESOM2/core2/'               ! path to mesh files (nod2d.out, elem2d.out, etc.)
+ClimateDataPath  = '/albedo/pool/FESOM2/hydrography/phc3.0/'  ! path to initial conditions (temperature, salinity)
+ResultPath       = '<add path>'                               ! path for output files and fesom.clock file
```

Comment:
I think Resultpath must remain meaningful as it was otherwise the CI test seems to fail !